docker/cluster-test/cluster-test.Dockerfile

```diff
@@ -18,7 +18,9 @@ RUN rustup install $(cat rust-toolchain)
 COPY . /libra
 RUN docker/cluster-test/compile.sh
 FROM debian:buster
-RUN apt-get update && apt-get install -y openssh-client curl && curl -L "https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl" -o /usr/local/bin/kubectl && chmod +x /usr/local/bin/kubectl
```

Comment:
Thanks for breaking this long line.  Can we also separate the kube version into its own line? It's easier to read.

Reply:
Done